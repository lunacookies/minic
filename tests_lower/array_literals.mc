func main {
	empty := []
	one := [1]
	multi := [1, 2, 3]
	nested := [[1, 2], [3, 4], [5, 6], [7]]
	very_nested := [[[[[[]]]]]]
}
