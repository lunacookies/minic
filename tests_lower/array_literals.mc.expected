func main
	var empty [0]void
	var one [1]i64
	var multi [3]i64
	var nested [4][2]i64
	var very_nested [1][1][1][1][1][0]void
	{
		set empty = []
		set one = [1]
		set multi = [
			1,
			2,
			3,
		]
		set nested = [
			[
				1,
				2,
			],
			[
				3,
				4,
			],
			[
				5,
				6,
			],
			<missing>,
		]
		set very_nested = [[[[[[]]]]]]
	}
tests_lower:93..96: error: expected “[2]i64” but found “[1]i64”
tests_lower:13..24: warning: unused variable
tests_lower:26..36: warning: unused variable
tests_lower:38..56: warning: unused variable
tests_lower:58..97: warning: unused variable
tests_lower:99..126: warning: unused variable
